Documentation/ApiOverview/Backend/JavaScript/ES6/Index.rst

```diff
@@ -6,29 +6,111 @@
 ES6 in the TYPO3 Backend
 ========================
 
-..  todo: Add information on ES6 and its registration
-    https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/5
-
-..  seealso::
-    This chapter is still work in progress. For now see the changelog:
+..  versionchanged:: 12.0
+    Starting with TYPO3 v12.0 JavaScript ES6 modules may be used instead of
+    AMD modules, both in backend and frontend context.
```

Comment:
Why mentioning frontend? I can use it also in v11 in frontend if I want. It sounds, that only v12 is capable of using ES6 in frontend - which is not true.

Reply:
It is written like that in the changelog entry